crates/host/src/module.rs

```diff
@@ -14,21 +13,30 @@ use std::io::Write;
 use std::path::PathBuf;
 use std::str::FromStr;
 use std::sync::Arc;
+#[cfg(feature = "wasmer_sys")]
```

Comment:
I feel like this is messy. Not sure if there is a preferred way to organize these feature flagged code

Reply:
If you just want less cfgs you can reexport from a private module, like:

```rust
#[cfg(feature = "wasmer_sys")]
mod sys_imports {
    //use things...;
}
#[cfg(feature = "wasmer_sys")]
use sys_imports::*;
```

Reply:
I'm not too worried about the `use`s, but after reading through this host section and the `test/wasms.rs` below, I think it might be worth doing the features as completely separate modules that expose a consistent api. Something like:

```
src/module.rs
src/module/wasmer_sys.rs
src/module/wasmer_wamr.rs
```

This might be easier to reason abut.

Reply:
split out feature-flagged code into separate files as suggested